docs/source/customizing.rst

```diff
@@ -0,0 +1,90 @@
+
+Customizing nbconvert
+=====================
+
+Look at the first 20 lines of the ``python`` exporter
+
+.. code:: python
+
+    pyfile = !jupyter nbconvert --to python 'Index.ipynb' --stdout
+    for l in pyfile[20:40]:
+        print l
```

Comment:
Things like this will really want the output in the converted result as well as the code, I guess. Might need a custom template!